aten/src/ATen/test/stride_properties_test.cpp

```diff
@@ -69,10 +69,25 @@ TEST(StridePropertiesTest, ZeroStrideIndicesEagerConsistencyTest) {
 }
 
 TEST(StridePropertiesTest, ExpandedStrideIndicesTest) {
-  // NOLINTNEXTLINE(performance-for-range-copy)
   Tensor t = at::rand({1});
   // note: expand with dimension of size 1 is tricky as stride is different
   // depending on the order of the unsqueezed dimension.
   t = t.expand({4, 4, 4});
   EXPECT_TRUE(CheckStrideIndices(t, at::MemoryFormat::Contiguous));
 }
+
+TEST(StridePropertiesTest, SlicedStrideIndicesTest) {
+  // Sliced tensor shouldn't have changed stride order
+  Tensor t = at::rand({16, 4}).slice(1, 0, 4, 4);
+
+  auto temp = TensorType::create(c10::nullopt, c10::nullopt, t.sizes(), t.strides(), c10::nullopt);
+  TORCH_INTERNAL_ASSERT(temp->stride_properties().isComplete() &&
+      temp->stride_properties().isComplete(), "complete stride properties is needed for the test");
+  std::vector<size_t> stride_indices(2);
+  std::iota(stride_indices.rbegin(), stride_indices.rend(), 0);
+
+  auto index_iter = stride_indices.begin();
+  for (const auto& opt_stride : *temp->stride_properties().sizes()) {
+    EXPECT_TRUE(*index_iter++ != opt_stride->stride_index_.value());
```

Comment:
this is failing ... which I think makes sense @jjsjann123

Reply:
errr... I was dumb.... That's a copy-paste went wrong....